src/components/WorkshopPage/Workshop.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import ReactPlayer from 'react-player/youtube';
+import Container from '@material-ui/core/Container';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import useTheme from '@material-ui/core/styles/useTheme';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+
+
+const workshops = [
+	{
+		type: 'Web Development',
+		elements: [
+			{
+				title: 'Intro to HTML',
+				author: 'Eric',
+				description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit,',
+				url: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ&ab_channel=RickAstleyVEVO'
+			},
+			{
+				title: 'Intro to CSS',
+				author: 'Chandra',
+				description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit,',
+				url: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ&ab_channel=RickAstleyVEVO'
+			},
+			{
+				title: 'Intro to HTML',
```

Comment:
```suggestion
				title: 'Intro to Something Else',
```

React currently emits a warning:

```
Warning: Encountered two children with the same key, `Intro to HTML`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.
```